mountinfo/mounted_linux.go

```diff
@@ -87,15 +89,26 @@ func mountedFast(path string) (mounted, sure bool, err error) {
 }
 
 func mounted(path string) (bool, error) {
-	path, err := normalizePath(path)
+	normalizedPath, err := normalizePath(path)
+	if err != nil && errors.Is(err, fs.ErrNotExist) {
+		if filepath.IsAbs(path) {
```

Comment:
This is primarily because we do not know where the ENOENT is coming from; I am not sure I like this as much as I want to :/